### ISSUE TITLE
Show popularity of templates and commands

### ISSUE THREAD

**Issue by jithine** [screwdriver-cd/screwdriver#1091]
## Context

With ever increasing number of commands & templates, users need means to know which ones are popular and in heavy usage. This helps users in picking up the appropriate template based on usage of their peers.

## Objective

1. Find a metric to track for commands. 
1. Find a metric to track for templates. 
1.1 Metric can be stars, or usage statistics
1. Display in UI a metric which show the popularity of a template.
1. Display in UI a metric which show the popularity of a command.

## References

1. Templates https://docs.screwdriver.cd/user-guide/templates
1. Commands https://docs.screwdriver.cd/user-guide/commands

**Comment by jithine** [comment]
One problem with current implementation is that template information is not stored in job/build model. If template name was stored then we can easily query for number of builds using the template

**Comment by tkyi** [comment]
Results:
<img width="1680" alt="Screen Shot 2020-02-14 at 11 59 53 AM" src="https://user-images.githubusercontent.com/3230529/74563409-9923d980-4f21-11ea-8c83-36339dd0e833.png">

**Comment by DekusDenial** [comment]
@InderH If I stop a job from using a template, will the `Jobs` counter go down by 1?

**Comment by InderH** [comment]
@DekusDenial yes the Jobs counter should go down, however, it looks like this is currently not the case. PR to fix this: https://github.com/screwdriver-cd/models/pull/436

**Comment by kumada626** [comment]
Currently, if a job that uses template is archived, `Jobs` counter does not go down.
I think the `Jobs` counter should go down in such case.
